.travis.yml

```diff
@@ -17,4 +17,6 @@ script:
   - git update-index -q --refresh
   - git diff-index --exit-code HEAD
   # Check that Gw only test passes (full test not supported yet in travis)
-  - sudo ./tests/test_gw_repeater.sh --rm --gateway-only 
+  - sudo ./tests/test_gw_repeater.sh --rm --gateway-only
+  # Check tlvf test
+  - ./run.sh --entrypoint "$PWD/../../../build/install/bin/tests/tlvf_test"
```

Comment:
I apparently missed this commit during my review, because I would have suggested a different approach: add a test target to maptools, and execute it by default when building. The unit tests take just a few seconds, so there is no reason not to run them I think.